src/Point/PointToInterlockingConnectionB4R1Impl.cs

```diff
@@ -0,0 +1,153 @@
+using EulynxLive.Messages.Baseline4R1;
+using EulynxLive.Point;
+using Google.Protobuf;
+using Grpc.Core;
+using Grpc.Net.Client;
+using Sci;
+using EulynxLive.Messages.IPointToInterlockingConnection;
+using static EulynxLive.Messages.IPointToInterlockingConnection.IPointToInterlockingConnection;
+using static Sci.Rasta;
+
+public class PointToInterlockingConnectionB4R1Impl : IPointToInterlockingConnection
```

Comment:
Readability of this name is not ideal. Adding the Impl suffix is usually unnecessary in .NET, since interfaces are distinguished by the `I` prefix. Suggestion: move this to namespace EulynxLive.Point.EulynxBaseline4R1 and call it PointToInterlockingConnection?